test/TritonGPU/metaws-loop-schedule.mlir

```diff
@@ -0,0 +1,196 @@
+    %off_z = arith.divsi %off_hz, %H : i32
+    %off_h = arith.remsi %off_hz, %H : i32
+    %offset_y = arith.muli %N_CTX, %H : i32
+    %offset_y_18 = arith.muli %off_z, %offset_y : i32
+    %offset_y_19 = arith.muli %off_h, %N_CTX : i32
+    %offset_y_20 = arith.addi %offset_y_18, %offset_y_19 : i32
+    %qo_offset_y = arith.muli %start_m, %c128_i32 : i32
+    %qo_offset_y_21 = arith.addi %offset_y_20, %qo_offset_y : i32
+    %offs_m = tt.make_range {end = 128 : i32, start = 0 : i32} : tensor<128xi32, #ttg.slice<{dim = 1, parent = #blocked}>>
+    %offs_m_22 = tt.make_range {end = 128 : i32, start = 0 : i32} : tensor<128xi32, #blocked2>
+    %offs_m_23 = tt.splat %qo_offset_y : i32 -> tensor<128xi32, #ttg.slice<{dim = 1, parent = #blocked}>>
+    %offs_m_24 = tt.splat %qo_offset_y : i32 -> tensor<128xi32, #blocked2>
+    %offs_m_25 = arith.addi %offs_m_23, %offs_m : tensor<128xi32, #ttg.slice<{dim = 1, parent = #blocked}>>
+    %offs_m_26 = arith.addi %offs_m_24, %offs_m_22 : tensor<128xi32, #blocked2>
+    %qk_scale = arith.mulf %sm_scale, %cst : f32
+    %q = tt.descriptor_load %desc_q[%qo_offset_y_21, %c0_i32] : !tt.tensordesc<tensor<128x128xf16, #shared>> -> tensor<128x128xf16, #blocked3>
+    %q_27 = ttg.local_alloc %q : (tensor<128x128xf16, #blocked3>) -> !ttg.memdesc<128x128xf16, #shared, #smem>
+    %m_ij = tt.splat %qk_scale : f32 -> tensor<128xf32, #ttg.slice<{dim = 1, parent = #blocked}>>
+    %qk = tt.splat %qk_scale : f32 -> tensor<128x64xf32, #blocked>
+    %qk_28, %qk_29 = ttng.tmem_alloc : () -> (!ttg.memdesc<128x64xf32, #tmem, #ttng.tensor_memory, mutable>, !ttg.async.token)
+    %acc_30, %acc_31 = ttng.tmem_alloc : () -> (!ttg.memdesc<128x128xf32, #tmem1, #ttng.tensor_memory, mutable>, !ttg.async.token)
+    %acc_32 = ttng.tmem_store %acc, %acc_30[%acc_31], %true : tensor<128x128xf32, #blocked1> -> !ttg.memdesc<128x128xf32, #tmem1, #ttng.tensor_memory, mutable>
+    // CHECK: scf.for {{.*}}
+    %offsetv_y:6 = scf.for %offsetv_y_56 = %c0_i32 to %qo_offset_y step %c64_i32 iter_args(%l_i_57 = %l_i, %m_i_58 = %m_i, %offset_y_59 = %offset_y_20, %arg29 = %false, %qk_60 = %qk_29, %acc_61 = %acc_32) -> (tensor<128xf32, #ttg.slice<{dim = 1, parent = #blocked}>>, tensor<128xf32, #ttg.slice<{dim = 1, parent = #blocked}>>, i32, i1, !ttg.async.token, !ttg.async.token)  : i32 {
+      // CHECK: tt.descriptor_load {{.*}} {loop.cluster = 1 : i32, loop.stage = 0 : i32} {{.*}}
+      %k = tt.descriptor_load %desc_k[%offset_y_59, %c0_i32] {tt.latency = 1 : i32} : !tt.tensordesc<tensor<64x128xf16, #shared>> -> tensor<64x128xf16, #blocked3>
+      %k_62 = ttg.local_alloc %k : (tensor<64x128xf16, #blocked3>) -> !ttg.memdesc<64x128xf16, #shared, #smem>
+      %k_63 = ttg.memdesc_trans %k_62 {order = array<i32: 1, 0>} : !ttg.memdesc<64x128xf16, #shared, #smem> -> !ttg.memdesc<128x64xf16, #shared1, #smem>
+      // CHECK: ttng.tc_gen5_mma {{.*}} {loop.cluster = 1 : i32, loop.stage = 0 : i32, tt.self_latency = 1 : i32} {{.*}}
```

Comment:
This stage information looks wrong. I'm going to investigate this in a followup.